File: python/arreglos/TercerEjercicio.py
import random
numeros = [0]*10
for i in range (10):
    numeros[i]= random.randint(1,100)
print("Arreglo original:")
print(numeros)
for i in range (len(numeros)):
    if numeros [i] % 2 !=0:
        numeros[i]=0
print("Impares vueltos ceros")
print(numeros)
for i in range(len(numeros)):
    numeros[i]=numeros[i]*i
print("Numeros multiplicados por su indice")
print(numeros)

File: C#/arreglos/TercerEjercicio.cs
using System;
class TercerEjercicio
{
    static void Main()
    {
        int [] numeros = new int [10];
        Random r = new Random();
        for(byte i = 0; i < numeros.Length; i++)
        {
            numeros[i]=r.Next(1,101);

        }
        Console.WriteLine("Arreglo original");
        foreach(int numero in numeros)
        {
            Console.WriteLine(numero);
        }
        for (byte i = 0; i < numeros.Length; i++)
        {
            if (numeros[i] % 2 != 0)
            {
                numeros[i]=0;
            }
        }
        Console.WriteLine("Arreglo modificado para impares vueltos ceros");
        foreach(int numero in numeros)
        {
            Console.WriteLine(numero);
        }
        for (byte i=0; i < numeros.Length; i++)
        {
            numeros[i]=numeros[i]*i;
        }
        Console.WriteLine("Arreglo modificado para multiplicar el numero por indice");
             foreach (int numero in numeros)
        {
            Console.WriteLine(numero);
        }
    
    }
}